lib/rake/ci.rake

```diff
@@ -81,12 +81,13 @@ namespace :ci do
     end
   end
 
-  task all: [
-    'firebase:ci',
-    :build_with_cloudfront,
-    :deploy_multi,
-    :publish_github_release
-  ]
+  all_tasks = []
+  all_tasks << 'firebase:ci'
+  all_tasks << :build_with_cloudfront
+  all_tasks << :deploy_multi
+  all_tasks << :publish_github_release if rack_env?(:production)
```

Comment:
Leaving it for now.  I considered this but think I prefer keeping this modeled on [build.rake](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/rake/build.rake#L136-L142), esp. with regard to what edits to this sequence might look like down the road.